.gitignore

```diff
@@ -21,6 +21,9 @@ builddir
 
 # i18n files
 po/lutris.pot
+po/POTFILES
+po/LINGUAS
+po/*.mo
```

Comment:
`.mo` files won't be generated in the `po` dir.

Reply:
> `.mo` files won't be generated in the `po` dir.

If you use `poedit` or other tools to edit the `.po` file, the corresponding `.mo` will be generated in this directory by default. Although I can set the software to stop the generating, but I just want to give myself a convenience. Others may also get help.

Reply:
Hm that's weird. But I guess it doesn't hurt.